common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -140,6 +140,14 @@ class CapaFields(object):
         scope=Scope.settings,
         default=default_reset_button
     )
+    disable_check_button_without_answer = Boolean(
+        display_name=_("Disable Check Button Without Answer"),
+        help=_(
+            "Enter true or false. Make it impossible to click \"check\"/\"final check\" without selecting a choice."
```

Comment:
Nitpick: single quoted strings and double quoted strings are identical in Python, except for which character needs to be escaped (the single quote or the double quote). Since you're using double quotes in this string, it would be more readable to surround it with single quotes, like this:
```
'Enter true or false. Make it impossible to click "check"/"final check" without selecting a choice.'
```